deal.II-toolchain/platforms/supported/sierra.platform

```diff
@@ -0,0 +1,33 @@
+# macOS 10.12 (Sierra)
+#
+# macOS support is experimental and requires:
+# - a recent version of xcode with command line tools
+# - openmpi installed using homebrew and
+# - cmake
+# 
+# Detailed Installation Notes:
+# - install Xcode from AppStore, open it and accept the license.
+# - Open Terminal, and install Xcode command line tools via
+# $ xcode-select --install
+# and then run
+# $ xcodebuild -license
+# to accept the license.
+# 
+# Install Homebrew in a Terminal via
+# $ /usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
+# 
+# Install the following via Homebrew
+# $ brew install cmake
+# $ brew install openmpi --c++11 --with-mpi-thread-multiple
```

Comment:
I'm not an expert in brew, but from their name I would like to have c++11 support (needed in my software) and also threads within a mpi process.
Maybe c++11 support is handled by the underlying apple clang anyway, I don't know.